app/views/content/is-teaching-right-for-me/teaching-internship-providers.md

```diff
@@ -15,8 +16,10 @@ quote_list:
 backlink: /
 fullwidth: true
 content:
-  - content/teaching-internship-providers/listing
-
+  - content/is-teaching-right-for-me/teaching-internship-providers/listing
+navigation: 5.40
+navigation_title: What is a teaching internship?
```

Comment:
Is this the right question? Should it be more like 'Can I get a teaching internship'?

Happy for you to disagree!